pkg/snapshot/velero.go

```diff
@@ -388,9 +389,16 @@ func getVeleroPod(ctx context.Context, clientset *kubernetes.Clientset, namespac
 		"component": "velero",
 		"deploy":    "velero",
 	}
+	labelSelector := labels.SelectorFromSet(veleroLabels)
+
+	if util.IsEmbeddedCluster() {
+		labelSelector = labels.SelectorFromSet(map[string]string{
+			"name": "velero",
+		})
+	}
```

Comment:
embedded cluster installs the velero chart, which sets the `name` labels by default on velero and node-agent

Reply:
and it's not possible to pass additional labels to the velero chart?

Reply:
you can, but they are all applied to both the velero and node-agent pods from what I was seeing (the `podLabels` value specifically). you can add labels to the deployment/daemonset, but we query for pods in KOTS